File: Server/Server/Models/Patient.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Server.Models
{
    public class Patient
    {
        public string idpatient { get; set; }
        public string name { get; set; }
        public string email { get; set; }
        public DateTime birthdate { get; set; }
        public string phonenumber { get; set; }
        public string location { get; set; }
        public Double height { get; set; }
        public Double weight { get; set; }
        public string password { get; set; }

    }
}

File: Application (Android Studio)/Mycure/app/src/main/java/com/example/mycure/classes/Patient.kt
package com.example.mycure.classes

import org.json.JSONObject
import java.io.Serializable

data class Patient(val fullName: String, val identification:String, val birthdate:String,
val location:String, val phoneNumber:String, val email:String, val height:String, val weight:String)
    :Serializable{
    constructor(json:JSONObject):this(
        json.getString("name"),
        json.getString("idpatient"),
        json.getString("birthdate"),
        json.getString("location"),
        json.getString("phonenumber"),
        json.getString("email"),
        json.getString("height"),
        json.getString("weight")
    )
}